.github/workflows/connect-nextra-dev-release.yml

```diff
@@ -7,8 +7,16 @@ permissions:
 on:
   pull_request:
     paths:
+      - "packages/connect-explorer-theme/**"
       - "packages/connect-explorer-nextra/**"
       - ".github/workflows/connect-nextra-dev-release.yml"
+  push:
+    paths:
+      - "packages/connect-explorer-theme/**"
+      - "packages/connect-explorer-nextra/**"
+      - ".github/workflows/connect-nextra-dev-release.yml"
+    branches:
+      - develop
```

Comment:
This could be on `schedule` like we are doing on the others actions:

```yml
  schedule:
    # Runs at midnight UTC every day at 01:00 AM CET
    - cron: "0 0 * * *"
```

So in develop we run everything and build it during the nightly schedule. 
The main reason to run in develop is to find out issues that are not considered in the defined paths. Otherwise if we run it in the branch and we run the same in develop there is no any added value.

Reply:
Ok, I'll change it